views/admin/admin-page.php

```diff
@@ -7,35 +7,245 @@
+										</label>
+										<span class="form-toggle__switch"></span>
+									</label>
+								</span>
+							</div>
+						</div>
+					</div><?php // j-row ?>
+
+				</div> <?php // nux-in ?>
+			</div><?php // j-col ?>
+		<?php // END WordPress.com Tools ?>
+
+	</div><?php // j-row ?>
+
+		<p><?php _e( 'Jetpack includes many other features that you can use to customize how your site looks and functions. These include Contact Forms, Tiled Photo Galleries, Custom CSS, Image Carousel, and a lot more.', 'jetpack' ); ?></p>
+
+		<?php if ( current_user_can( 'jetpack_manage_modules' ) ) : ?>
+			<p><a href="<?php echo admin_url( 'admin.php?page=jetpack_modules' ); ?>" class="button full-features-btn" ><?php echo sprintf( __( 'See the other %s Jetpack features', 'jetpack' ), count( Jetpack::get_available_modules() ) - count( $data['recommended_list'] ) ); ?></a></p>
+		<?php endif; ?>
+
+		<div class="nux-foot j-row">
+			<div class="j-col j-lrg-9 j-md-9 j-sm-12">
+			<?php
+				// Get a list of Jetpack Happiness Engineers.
+				$jetpack_hes = array(
+					'724cd8eaaa1ef46e4c38c4213ee1d8b7',
+					'623f42e878dbd146ddb30ebfafa1375b',
+					'561be467af56cefa58e02782b7ac7510',
+					'd8ad409290a6ae7b60f128a0b9a0c1c5',
+					'790618302648bd80fa8a55497dfd8ac8',
+					'6e238edcb0664c975ccb9e8e80abb307',
+					'4e6c84eeab0a1338838a9a1e84629c1a',
+					'9d4b77080c699629e846d3637b3a661c',
+					'4626de7797aada973c1fb22dfe0e5109',
+					'190cf13c9cd358521085af13615382d5',
+				);
+
+				// Pick a random HE.
+				$jetpack_he = array_rand( $jetpack_hes, 1 );
+
+				// Get a random profile URL.
+				$default_he_img = plugins_url( 'images/jetpack-icon.jpg', JETPACK__PLUGIN_FILE );
+
+				printf(
+					'<a href="http://jetpack.me/support/" target="_blank"><img src="https://secure.gravatar.com/avatar/%1$s?s=75&d=%2$s" alt="Jetpack Happiness Engineer" /></a>',
+					$jetpack_hes[ array_rand( $jetpack_hes ) ],
+					urlencode( $default_he_img )
+				);
+			?>
+			<p><?php _e( 'Need help? The Jetpack team is here for you!', 'jetpack' ); ?></p>
+			<p><?php
+				$jetpack_support_url = sprintf(
+					'<a href="http://jetpack.me/support/" target="_blank" title="%1$s">%1$s</a>',
+					esc_attr__( 'View our support page', 'jetpack' )
+				);
+
+				$jetpack_forum_url = sprintf(
+					'<a href="https://wordpress.org/support/plugin/jetpack" target="_blank" title="%1$s">%1$s</a>',
+					esc_attr__( 'check the forums for answers', 'jetpack' )
+				);
+
+				$jetpack_contact_url = sprintf(
+					'<a href="http://jetpack.me/contact-support/" target="_blank" title="%1$s">%1$s</a>',
+					esc_attr__( 'contact us directly', 'jetpack' )
+				);
+
+				printf(
+					_x(
+						'We offer free, full support to all of our Jetpack users. Our support team is always around to help you. %1$s, %2$s, or %3$s',
+						'1: View our support page; 2: check the forums for answers, 3: contact us directly',
```

Comment:
You build the links about, but you don't actually use them in the string here...